task-sdk/tests/task_sdk/bases/test_hook.py

```diff
@@ -101,7 +101,7 @@ def test_get_connection_secrets_backend_configured(self, mock_supervisor_comms,
         path = tmp_path / "conn.env"
         path.write_text("CONN_A=mysql://host_a")
 
-        with conf_vars(
+        with task_sdk_conf_vars(
```

Comment:
See a previous comment I just left, I think this one should not exist and

```suggestion
        with conf_vars(
```

should be made to Do The Right Thing.

Reply:
Handled via: 4a0e96022a9123035ca2ea247422300a635ed926